State/Stats.cpp

```
@@ -391,9 +401,11 @@ void Stats::formatVideoStats(DX::StepTimer const& timer, VIDEO_STATS& stats, cha
 					   "------\n"
 					   "Missed present rate: %.2f%%\n"
 					   "PreWait/Render: %.2f/%.2f ms\n",
+					   "GPU render cost min/max/avg: %.2f/%.2f/%.2f ms\n"
 					   stats.hitDeadlines ? ((double)stats.missedDeadlines / (stats.missedDeadlines + stats.hitDeadlines)) * 100 : 0.0f,
```

Cppcheck (2.21.0)

[error] Syntax Error

(internalAstError)